test/linters/accessibility/no_redundant_image_alt_test.rb

```diff
@@ -28,4 +27,14 @@ def test_does_not_warn_if_alt_contains_no_redundant_text
 
     assert_empty @linter.offenses
   end
+
+  def test_does_not_warn_if_linter_is_disabled_in_file
+    @file = <<~HTML
+      <%# erblint:disable GitHub::Accessibility::NoRedundantImageAlt %>
+      <img alt='image of an octopus'></img>
+    HTML
```

Comment:
Our linters have logic that allows a linter to be disabled/ignored in a file. This comes from: `rule_disabled?`

Right now, we only check the demodulized name so:
```
<%# erblint:disable NoRedundantImageAlt %>
```
and a totally invalid name like:
```
<%# erblint:disable BadModule::NoRedundantImageAlt %>
```
can be used to disable a linter rule in a file.


I updated [this line](https://github.com/github/erblint-github/pull/2/files#diff-68805d412cb4e6c133922148c362d224c0148a47c21347e3f7b4a23907b690e8R14), to require the fuller linter name (not including `ERBLint::Linters` bcuz that is redundant). 

Our linter names are namespaced and this seems  more conflict proof!